charts/gateway-helm/values.tmpl.yaml

```diff
@@ -26,6 +26,10 @@ deployment:
     labels: {}
 
 config:
```

Comment:
can this be solved by setting `config: null` via values and using `{{- if .Values.config }}` in the template ?

Reply:
we could expose the configMap `name` ( defaults to `envoy-gateway-config`) using `config.name`

Reply:
another way to arrange this is
```
config:
  envoyGateway:
    name: ........
    value: ........
```

thoughts @Xunzhuo ?

Reply:
sure, we can set the default name to `envoy-gateway-config`, i have chosen to hardcore it in the templates as i have seen that this is the default name hardcoded in the controller logic. Maybe we want to split the fields and leave `config.envoyGateway` as is, and have separate
```
configmap:
  enabled: true
  nameOverride: ""
```